src/CentreonRemote/Application/Webservice/CentreonConfigurationRemote.php

```diff
@@ -299,7 +299,11 @@ public function postLinkCentreonRemoteServer(): array
         if ($isRemoteConnection) {
             $serverConfigurationService->setDbUser($this->arguments['db_user']);
             $serverConfigurationService->setDbPassword($this->arguments['db_password']);
-            if ($serverWizardIdentity->checkBamOnRemoteServer($serverIP, $centreonPath)) {
+            if ($serverWizardIdentity->checkBamOnRemoteServer(
+                $httpMethod . '://' . $serverIP . ':' . $httpPort . trim($centreonPath, '/'),
```

Comment:
`trim()` is incorrect here, as it will remove the `/` char between the port number and the Centreon path.
Removing it makes this PR to flawlessly work :+1:

Reply:
I fix it